caf-audit/src/main/java/com/hpe/caf/auditing/elastic/ElasticAuditEventBuilder.java

```diff
@@ -97,7 +99,8 @@ public void setTenant(String tenantId){
         }
         this.tenantId = tenantId.toLowerCase();
 
-        // TODO CAF-2610 - Create Elasticsearch index for the specified tenant if it does not already exist.
+        // Create Elasticsearch index for the specified tenant.
+        indexManager.getIndex(ES_INDEX_PREFIX.concat(this.tenantId));
```

Comment:
@glucy A little confused around this one to be honest. Right now the ES index we create is "audit_tenant_<tenantId>". What should the full index name be right now for a tenant? Keep in mind an index in ES cannot begin with an underscore.